src/Raven.Server/ServerWide/ClusterStateMachine.License.cs

```diff
@@ -289,13 +290,16 @@ private void AssertRevisionConfiguration(DatabaseRecord databaseRecord, LicenseS
 
         foreach (KeyValuePair<string, RevisionsCollectionConfiguration> revisionPerCollectionConfiguration in databaseRecord.Revisions.Collections)
         {
+            if (revisionPerCollectionConfiguration.Value.Disabled)
```

Comment:
you are allowing to insert values (disabled) not should not be allowed. What are the implications when you will enable them? I do not see a test
This impacts the 'GetRevisionsConfiguration' method, wouldn't it impact 'ShouldVersionDocument' ?